app/models/person.rb

```diff
@@ -66,4 +66,14 @@ def for_feed
   def latest_person_escort_record
     person_escort_records.where(status: 'confirmed').order(confirmed_at: :desc).first
   end
+
+  def category
+    if latest_nomis_booking_id.present?
+      @category ||= begin
```

Comment:
I've not been involved in the discussions around why we want to expose the category on a person in realtime vs using profiles (which might not be up to date with the latest category as that represents those at that specific time), can you walk me through which dashboards/process we need to pull in this value for?

Reply:
The use case is that the front end can call `GET /people/:id` endpoint to verify that the prisoner is not in a high risk category and should be prevented from being moved; this can happen early in the UX journey rather than the current approach of raising an error when the new move is created in the backend.

We always want to check the latest prisoner category in nomis for this use case.. however, it would seem reasonable to store the `category_id` in the profile though once we've fetched this data so it can be useful for reporting or other less critical endpoints.

Reply:
Yes sure:
When creating a move it would be good to know in advance of creating the move and profile records whether the person being moved is a Cat-A prisoner, and if so don't allow the move. We can't check the profile record because it doesn't yet exist when the move is initiated.

From a UI point of view, my thinking is: 
1) search for the person to be moved
2) select the person record from the list presented
3) check whether they can be moved by calling `GET /people/id?include=category` which does a Nomis lookup
4) if they can be moved, then continue to as normal to fill in the PTR. Otherwise exist with a message about not moving Cat-A prisoners.

Reply:
To confirm, we do store category in the profile record. At the moment this is an attribute, `category_code`. In a separate PR I will migrate this to `category_id` (ideally without upsetting the JPC reports).

Also, to confirm: as discussed offline, we will not store category at the person level, only at the profile level.